nvOC

```diff
@@ -210,16 +210,51 @@ Reboot()
    sudo su -c "shutdown -r now"
 }
 
+
+############################### Reboot host ###################################
+Reboot()
+{
+   # Reboots the host
+   echo -e "${B}${R}Warning: The host will reboot in 10 seconds, ctrl-c to cancel...${N}"
+   sleep 10
+   sudo su -c "shutdown -r now"
+}
+############################### Reboot host ###################################
+Restart()
+{
+   # Reboots the host
+        nvoc stop && timeout 2 && nvoc start
```

Comment:
This script is called `nvOC` and filenames are of course case-sensitive. You could either get the actual path/filename from command line, or simply call the two Start() & Stop() functions in this same script.

Reply:
nvoc stop && nvoc start
Try it, works perfectly fine in terminal.